### PR TITLE
DOCSP-41967: Insert documents

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41967
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41967-insert/write/insert/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write/insert.txt [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+In a MongoDB collection, each document *must* contain an ``_id`` field
+with a unique field value.
+
+MongoDB allows you to manage this field in two ways:
+
+- You can set this field for each document yourself, ensuring each
+  ``_id`` field value is unique.
+- You can let the driver automatically generate unique ``ObjectId``
+  values for each document ``_id``.
```
Comment (jordan-smith721) [inline source/write/insert.txt:59]: S: Simplify
```suggestion
- Set it for each document yourself, ensuring each
  ``_id`` field value is unique.
- Let the driver automatically generate unique ``ObjectId``
  values for each document ``_id``.
```

File: source/write/insert.txt [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    :language: php
+    :dedent:
+    :start-after: start-insert-one
+    :end-before: end-insert-one
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, call the ``MongoDB\Collection::insertMany()``
+method and pass an array that stores the documents you want to add.
```
Comment (jordan-smith721) [inline source/write/insert.txt:94]:
```suggestion
method and pass an array that contains the documents you want to add.
```

File: source/write/insert.txt [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``bypassDocumentValidation``
+     - | If set to ``true``, allows the write to opt out of
```
Comment (jordan-smith721) [inline source/write/insert.txt:120]:
```suggestion
     - | If set to ``true``, allows the write operation to opt out of
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM + some smal suggestions

File: source/write/insert.txt [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+} to add
+documents to a MongoDB collection by performing **insert operations**.
+
+An insert operation inserts one or more documents into a MongoDB collection.
+You can perform an insert operation by using the ``MongoDB\Collection::insertOne()``
+method to insert a single document or the ``MongoDB\Collection::insertMany()``
+method to insert one or more documents.
```
Comment (GromNaN) [inline source/write/insert.txt:29]: Personal opinion: with a list it's easier to read.

```suggestion
You can perform an insert operation by using this ``MongoDB\Collection`` methods:
-  ``insertOne()`` to insert a single document
-  ``insertMany()`` to insert one or more documents.
```

File: source/write/insert.txt [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    :language: php
+    :dedent:
+    :start-after: start-insert-one
+    :end-before: end-insert-one
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, call the ``MongoDB\Collection::insertMany()``
+method and pass an array that contains the documents you want to add.
```
Comment (GromNaN) [inline source/write/insert.txt:94]:
```suggestion
method and pass an array that contains the list of documents you want to add.
```

File: source/write/insert.txt [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+.. literalinclude:: /includes/write/insert.php
+    :language: php
+    :dedent:
+    :start-after: start-modify
+    :end-before: end-modify
+
+Additional Information
+----------------------
+
+.. TODO:
```
Comment (GromNaN) [inline source/write/insert.txt:160]: Will this be addressed in a follow-up ticket?

Comment (norareidy) [reply source/write/insert.txt:160]: Yes, there will be a final cleanup ticket that resolves all the TODOs

File: source/write/insert.txt [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+--------
+
+In this guide, you can learn how to use the {+php-library+} to add
+documents to a MongoDB collection by performing **insert operations**.
+
+An insert operation inserts one or more documents into a MongoDB collection.
+You can perform an insert operation by using the following methods:
+
+- ``MongoDB\Collection::insertOne()`` to insert a single document
+- ``MongoDB\Collection::insertMany()`` method to insert one or more documents
```
Comment (GromNaN) [inline source/write/insert.txt:30]:
```suggestion
- ``MongoDB\Collection::insertMany()`` to insert one or more documents
```

**Pull request merged**